github-dark.css

```diff
@@ -1738,7 +1738,7 @@
   button.tabnav-tab, .rss-subscribe, a.rss-subscribe, table.capped-list .octicon,
   .site-header-menu .nav-item, .discussion-item-icon, .exploregrid-item,
   a.exploregrid-item, .commit-ref .user, .markdown-body blockquote strong {
-    color: #b5b5b5 !important;
+    color: #e2e2e2 !important;
```

Comment:
@auscompgeek see https://github.com/StylishThemes/GitHub-Dark/pull/364

not the perfect solotion though it fixes your bit but other parts remain far too dull.